__tests__/app.test.js

```diff
@@ -123,4 +123,43 @@ describe("NC News API testing", () => {
         });
     });
   });
+  describe("/api/articles/:article_id/comments", () => {
+    test("returns an array of comments with appropriate properties", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.comments).toEqual({
+            article_id: 1,
+            author: "icellusedkars",
+            body: "I hate streaming noses",
+            comment_id: 5,
+            created_at: "2020-11-03T21:00:00.000Z",
+            votes: 0,
+          });
+        });
+    });
```

Comment:
This should be an array of comments, not just 1 comment. Please see my comment in your model